alerter.py

```diff
@@ -1,25 +1,23 @@
-alert_failure_count = 0
+import alerter_stub
 
-def network_alert_stub(celcius):
-    print(f'ALERT: Temperature is {celcius} celcius')
-    # Return 200 for ok
-    # Return 500 for not-ok
-    # stub always succeeds and returns 200
-    return 200
+alert_failure_count = 0
 
 def alert_in_celcius(farenheit):
     celcius = (farenheit - 32) * 5 / 9
-    returnCode = network_alert_stub(celcius)
+    returnCode = alerter_stub.network_alert_stub(celcius)

```

Comment:
moving the test code to another file is just a logical step. But the main change has to come from production code. The production code should not be aware of test code. If the 3rd party component is not available during testing, then we abstract the component with an interface or function pointer and inject during execution.